src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/settings/main.js

```diff
@@ -377,6 +380,26 @@ define([
                 this.sandbox.dom.attr('#shadow_on_checkbox', 'checked', true);
                 this.sandbox.emit('husky.toolbar.header.item.disable', 'state', false);
             }
+
+            var def = $.Deferred();
+
+            // set author form data
+            this.sandbox.form.create('#settings-author-container').initialized.then(function() {
+                // the 'c' prefix is needed for the contact-selection
+                // which uses this to determine the type of id
+                // c for contact and a for account
```

Comment:
So the `'c'` prefix is here, although organization will never occur? (just a question for me to understand)